### DISCUSSION TITLE
Option for displaying time to read

### DISCUSSION THREAD

**Discussion by Backspaze** [GetPublii/Publii#1658] [Ideas]
I'd like to see an option for displaying time to read on posts/pages. There are many examples of this out there, but I've yet to find an implementation of this for Publii. There's already a reading time calculator in the editor, so I'm guessing it shouldn't be too hard to leverage that to add this to posts/pages as well?

**Comment by bobmitro** [comment]
This feature can be seamlessly integrated into the theme using a script.  There are many scripts on the market that measure the reading time of articles.

**Reply by Backspaze** [reply]
Ok, so it's a theme related feature, got it! I guess I'll have to take a look at the [dev docs](https://getpublii.com/dev/) and see if I can figure out how to add it.

**Reply by candidexmedia** [reply]
@Backspaze I'm curious about this too! Please share your findings/solutions, and let us know if you run into any issues. I've implemented many a custom JS library to my sites, and I'm curious to know what's out there for this.

**Comment by Backspaze** [comment]
There's now an official plugin for this - #2370.
